controllers/storagecluster/volumesnapshotterclasses.go

```diff
@@ -143,6 +144,12 @@ func (obj *ocsSnapshotClass) ensureCreated(r *StorageClusterReconciler, instance
 	cephFsSnapClass.snapshotClass.Name = util.GenerateNameForSnapshotClass(instance.Name, util.CephfsSnapshotter)
 	volumeSnapshotClasses := []SnapshotClassConfiguration{rbdSnapClass, cephFsSnapClass}
 
+	if instance.Spec.ExternalStorage.Enable {
```

Comment:
Why is there a need for a check. I believe that this code only runs in external mode